pkg/trace/config/subscriber.go

```diff
@@ -31,6 +32,19 @@ func (s *Subscriber) Get(req *pbgo.GetConfigsRequest) (*pbgo.ConfigResponse, err
 		return nil, errors.New("not allowed")
 	}
 	s.once.Do(s.subscribe)
+LOOP:
```

Comment:
```suggestion
loop:
```
No need for caps, I think. Although, I appreciate the old school label setting 🙂